test/e2e/traffic_split_test.go

```diff
@@ -77,7 +77,7 @@ func formatActualTargets(t *testing.T, actualTargets []string) (formattedTargets
 
 // TestTrafficSplit runs different e2e tests for service traffic splitting and verifies the traffic targets from service status
 func TestTrafficSplit(t *testing.T) {
-	t.Parallel()
+	//t.Parallel()
```

Comment:
Is there a reason for limiting parallelism... or just temporary?

Reply:
just temporary

Reply:
removed the commit